config.go

```diff
@@ -77,16 +237,94 @@ func loadLndConfig(preCfg *Config) (*lnd.Config, error) {
 		os.Exit(0)
 	}
 
+	// Load the main configuration file and parse any command line options.
+	// This function will also set up logging properly.
+	cfg, err := loadConfigFile(preCfg, usageMessage)
+	if err != nil {
+		return nil, err
+	}
+
+	// With the validated config obtained, we now know that the root logging
+	// system of lnd is initialized and we can hook up our own loggers now.
+	SetupLoggers(cfg.Lnd.LogWriter)
+
+	// Validate the lightning-terminal config options.
+	if cfg.LetsEncryptDir == "" {
+		cfg.LetsEncryptDir = filepath.Join(
+			cfg.Lnd.LndDir, defaultLetsEncryptDir,
```

Comment:
On my node, the `letsencrypt` dir is being placed in `~/.lnd/` instead of `~/.lit/`. Should this line join `cfg.LitDir` instead of `cfg.Lnd.LndDir`?

Reply:
Yes, definitely. Fixed.